src/main/kotlin/com/openlattice/mechanic/upgrades/AddPgAuditToExistingOrgs.kt

```diff
@@ -0,0 +1,54 @@
+package mechanic.src.main.kotlin.com.openlattice.mechanic.upgrades
+
+import com.openlattice.hazelcast.HazelcastMap
+import com.openlattice.mechanic.Toolbox
+import com.openlattice.mechanic.upgrades.Upgrade
+import com.openlattice.mechanic.upgrades.Version
+import com.openlattice.organizations.Organization
+import com.openlattice.postgres.external.ExternalDatabaseConnectionManager
+import org.slf4j.LoggerFactory
+
+/**
+ * @author Andrew Carter andrew@openlattice.com
+ */
+
+class AddPgAuditToExistingOrgs(
+    private val toolbox: Toolbox,
+    private val extDbConnMan: ExternalDatabaseConnectionManager
+
+) : Upgrade {
+
+    companion object {
+        private val logger = LoggerFactory.getLogger(AddPgAuditToExistingOrgs::class.java)
+        private const val PGAUDIT_EXTENSION = "pgaudit"
+        private const val OPENLATTICE_SCHEMA = "openlattice"
+    }
+
+    override fun upgrade(): Boolean {
+        val allOrgs = HazelcastMap.ORGANIZATIONS.getMap(toolbox.hazelcast).values.toList()
+
+        allOrgs.forEach { addPgAuditToOrg(it) }
```

Comment:
Please wrap the function call in a try-catch so that an issue with org does not fail the entire migration. In addition, what I like to do with migrations is 1. keep track of progress, and 2. use timers (helps with debugging). Here's a rough layout of how I have been doing it:

```
val organizations = HazelcastMap.ORGANIZATIONS.getMap(toolbox.hazelcast)
organizations.forEachIndexed { index, it ->
    try {
        logger.info("================================")
        logger.info("================================")
        logger.info("starting to process org ${it.key}")

        val timer = Stopwatch.createStarted()
        // do work

        logger.info("processing org took {} ms - org ${it.key}", timer.elapsed(TimeUnit.MILLISECONDS))
    }
    catch (e: Exception) {
        logger.error("something went wrong processing org ${it.key}", e)
    } finally {
        logger.info("progress ${index + 1}/${organizations.size}")
        logger.info("================================")
        logger.info("================================")
    }
}
```